packages/reassure-cli/src/constants.ts

```diff
@@ -0,0 +1,17 @@
+export const RESULTS_DIRECTORY = '.reassure';
+export const RESULTS_FILE = '.reassure/current.perf';
+export const BASELINE_FILE = '.reassure/baseline.perf';
+
+export const CHALK = {
+  colors: {
+    primary: '#4fe89a',
+    error: '#E73A0E',
+    warn: '#E7900E',
+    dim: '#919191',
+  },
+};
+
+export const GITIGNORE_DATA = `
```

Comment:
Can we put that in templates as gitignore file?

Reply:
done